src/modules/governance/models/proposal.votes.model.ts

```diff
@@ -0,0 +1,29 @@
+import { Field, ObjectType } from '@nestjs/graphql';
+
+@ObjectType()
+export class ProposalVotes {
+    @Field()
+    upVotes: string;
```

Comment:
why string? It seems these should be `number`

Reply:
done

Reply:
you have a regression here